ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -173,10 +215,26 @@ switch:
   evpn_route_servers: {{ switch_data.combined.evpn_route_servers | arista.avd.default() }}
 {%     endif %}
 
+
 {# switch.isis_net #}
 {%     if switch_data.combined.isis_system_id_prefix is arista.avd.defined %}
   isis_net: {{ isis_area_id }}.{{ switch_data.combined.isis_system_id_prefix }}.{{ '%04d' | format(switch_id) }}.00
 {%     endif %}
+
+{# switch.is_type #}
```

Comment:
add condition to check if `switch_underlay_routing_protocol` is one of the isis variants and set this accordingly. 
Implement the default like suggestion for underlay_routing_protocol.